moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_interface_base.h

```diff
@@ -55,6 +55,8 @@ namespace moveit_jog_arm
 class JogInterfaceBase
 {
 public:
+  JogInterfaceBase();
```

Comment:
It's somewhat weird that the constructor doesn't initialize the `planning_scene_monitor_` member.
But I see the simplicity of your approach...

Reply:
Expanded PR description